### PR TITLE
Check the file size on disk after the file transfer

### PR DESCRIPTION
**Pull request by lyw07**
### Summary

This PR is to fix the issue that the exercises are mistakenly considered as corrupted when transferring over local network. The issue behind that is we compare the size of the file over the network with the size in the database. Instead, we should compare the size of the file on the disk to the number in the database. Cherrypy compresses the files, so the size of the file over the network is smaller than the size in the database.

### Reviewer guidance

1. Check if we can successfully import exercises in `CK-12 > Math > Elementary Math > Grade 1 > Addition and Subtraction facts to 10` from local network.
2. Check that if a file on the external drive is corrupted, whether the file would be skipped. I tested this by removing a file and then creating a random file with the same name.

### References

https://github.com/learningequality/kolibri/issues/4790
https://community.learningequality.org/t/import-channel-content-from-network-location/985

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [X] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/core/content/management/commands/importcontent.py [hunk 1]
```diff
@@ -197,22 +197,25 @@ def _start_file_transfer(self, f, filetransfer, overall_progress_update):
...
                     file_dl_progress_update(length)
+
+                # If size of the destination file is smaller than the size
+                # indicated in the database, it's very likely that the source
+                # file is corrupted. Skip importing this file.
+                dest_file_size_on_disk = os.path.getsize(filetransfer.dest)
+                if dest_file_size_on_disk < f.file_size:
+                    e = "File {} is corrupted.".format(filetransfer.source)
+                    logger.error("An error occurred during content import: {}".format(e))
+                    os.remove(filetransfer.dest)
```
Comment (rtibbles) [inline kolibri/core/content/management/commands/importcontent.py:215]: Could we add an additional test to guarantee the cleanup behaviour also?

Comment (lyw07) [reply kolibri/core/content/management/commands/importcontent.py:215]: Yes I will do that!

Comment (lyw07) [reply kolibri/core/content/management/commands/importcontent.py:215]: I didn't add an additional test but test the behavior in an existing test for corrupted file. Please feel free to let me know if you think I need additional test only for the cleanup behavior. Thank you!

Comment (rtibbles) [reply kolibri/core/content/management/commands/importcontent.py:215]: Sounds good!

File: kolibri/core/content/management/commands/importcontent.py [hunk 2]
```diff
@@ -197,22 +197,25 @@ def _start_file_transfer(self, f, filetransfer, overall_progress_update):
...
+                    overall_progress_update(f.file_size - dest_file_size_on_disk)
```
Comment (rtibbles) [inline kolibri/core/content/management/commands/importcontent.py:216]: I feel like we need to make this correction to both progress processes, if the wire size is smaller than the size on disk?

**Pull request closed without merging**